janusgraph-dist/src/assembly/static/cassandra/bin/cassandra

```diff
@@ -82,27 +92,6 @@ elif [ -r "$CASSANDRA_INCLUDE" ]; then
     . "$CASSANDRA_INCLUDE"
 fi
 
-# Use JAVA_HOME if set, otherwise look for java in PATH
-if [ -n "$JAVA_HOME" ]; then
-    # Why we can't have nice things: Solaris combines x86 and x86_64
-    # installations in the same tree, using an unconventional path for the
-    # 64bit JVM.  Since we prefer 64bit, search the alternate path first,
-    # (see https://issues.apache.org/jira/browse/CASSANDRA-4638).
-    for java in "$JAVA_HOME"/bin/amd64/java "$JAVA_HOME"/bin/java; do
-        if [ -x "$java" ]; then
-            JAVA="$java"
-            break
-        fi
-    done
-else
-    JAVA=java
-fi
-
-if [ -z $JAVA ] ; then
-    echo Unable to find java executable. Check JAVA_HOME and PATH environment variables. >&2
-    exit 1;
-fi
-
```

Comment:
Why was this removed? I see `JAVA` variable is still used in this script.

Reply:
This file is mostly https://github.com/apache/cassandra/blob/trunk/bin/cassandra with some lines changed.

Reply:
I see it was removed here: https://github.com/apache/cassandra/commit/6ba2fb9395226491872b41312d978a169f36fcdb#diff-ca56a2b54ac847b9ff3908aac256aaa7f160680271725927a4b1917e64206011

Not sure why, but I guess that's fine